src/varianttur/varianttur.module.css

```diff
@@ -0,0 +1,212 @@
+:root {
+  --font-size-min: 12;
+  --font-size-max: 22;
+  --viewport-min: 400;
+  --viewport-max: 1500;
+}
+
+html {
```

Comment:
I didn't think Next.js allowed global classes in module files, do we have config that allows it or is it always allowed?

Reply:
It does when you override the css loader 😬 

From the next config:
```
module.exports = withPlugins([withTM, withImages], {
  target: "serverless",
  webpack: (config) => {
    const oneOf = config.module.rules.find(
      (rule) => typeof rule.oneOf === "object"
    );
    if (oneOf) {
      const moduleCssRule = oneOf.oneOf.find(
        (rule) => regexEqual(rule.test, /\.module\.css$/)
        // regexEqual(rule.test, /\.module\.(scss|sass)$/)
      );
      if (moduleCssRule) {
        const cssLoader = moduleCssRule.use.find(({ loader }) =>
          loader.includes("css-loader")
        );
        if (cssLoader) {
          cssLoader.options.modules.mode = "local";
        }
      }
    }
    return config;
  },
});
```

☝️ Finds files that are CSS modules and set module mode to local for the CSS loader.

Reply:
Just found that code 😁 Does that mean dl/dd/dt for instance only affects the `varianttur` page?

Reply:
Yes, it should only be included for that specific page.